bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "openemr",
+  "homepage": "http://open-emr.dev",
```

Comment:
No, that can be just OpenEMR. I believe that's more for if we were planning on publishing a package but when I ran 'bower init' it asked for a name so I put one in